server/game/cards/08.6-SAT/SaltcliffeSailor.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../drawcard.js');
+
+class SaltcliffeSailor extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.tokens['gold'] > 0,
+            effect: ability.effects.addKeyword('Stealth')
+        });
+        this.action({
+            title: 'Move 1 gold to character',
+            cost: ability.costs.discardGold(),
```

Comment:
This ability has no cost, as moving the token is solely part of the resolution. As you already modified the token on this card in the handler (so currently, 2 tokens would get removed to "create" a single new token on the other card) just getting rid of the cost property will work. But then you also need to make sure the ability cannot initiate if this card has no gold tokens, which is accomplished by

```javascript
condition: () => this.hasToken('gold')
```

The hasToken() shorthand method could also be used in the match function in the persistentEffect, but that doesn't matter much.